readthedocs/projects/tasks.py

```diff
@@ -1177,6 +1181,18 @@ def setup_python_environment(self):
         if self.project.has_feature(Feature.LIST_PACKAGES_INSTALLED_ENV):
             self.python_env.list_packages_installed()
 
+    def install_system_dependencies(self):
+        packages = self.config.build.apt_packages
+        if packages:
+            self.build_env.run(
+                'apt-get', 'update', '-y', '-q',
+                user=settings.RTD_BUILD_SUPER_USER,
+            )
+            self.build_env.run(
+                'apt-get', 'install', '-y', '-q', *packages,
```

Comment:
We should always use the long-version of arguments in code, to make it clearer what it does. Also add in the `--` Ending

```suggestion
                # -- ends all command arguments to protect against users passing them
                'apt-get', 'install', '--assume-yes', '--quiet', '--', *packages,
```

Reply:
Is there a reason we want this to be quiet? I know the output is sort of long but it might help folks debug their own problems. I tested it with `--quiet` and it's still fairly verbose but maybe a comment is appropriate on why.

I looked at what CircleCI does and they basically give users free reign to run arbitrary commands including `apt-get install`. Our system seems quite a bit more locked down than that which seems fine. There's still some risk (could a user install some docker utilities and somehow access the host?) but I think as long as we're just installing built-in Debian packages and sanitizing the inputs really well, we should be OK.

Reply:
> Is there a reason we want this to be quiet? I know the output is sort of long but it might help folks debug their own problems. I tested it with --quiet and it's still fairly verbose but maybe a comment is appropriate on why.

`-q` will suppress the progress bar not the output itself, `-qq` will suppress everything.